viewer/src/__tests__/types.ts

```diff
@@ -0,0 +1,12 @@
+/*!
+ * Copyright 2020 Cognite AS
+ */
+
+/**
+ * Retrieves the property type of a single property of a type.
+ */
+export type PropType<TObj, TProp extends keyof TObj> = TObj[TProp];
```

Comment:
I think there is something built-in in ts for that.

https://stackoverflow.com/a/41075862/2670121

Just tried it. `id` is a number so I got a type error as expected. 

![image](https://user-images.githubusercontent.com/4989157/100075401-2f394880-2e40-11eb-8890-effe4d2e5e03.png)

Reply:
Isn't that exactly what the `PropType` above does? I just feel it's a bit more easy to read, but it might just be me

Reply:
@larsmoa I don't mind you keep it if you like it. I just probably wouldn't ever use it, but it's totally up to you.